arch/rp2xx0/rp2350.ini

```
@@ -1,16 +1,15 @@
 ; Common settings for rp2040 Processor based targets
 [rp2350_base]
-platform = https://github.com/maxgerhardt/platform-raspberrypi.git#19e30129fb1428b823be585c787dcb4ac0d9014c ; For arduino-pico >=4.2.1
+platform = https://github.com/maxgerhardt/platform-raspberrypi.git#76ecf3c7e9dd4503af0331154c4ca1cddc4b03e5 ; For arduino-pico >= 4.4.3
 extends = arduino_base
-platform_packages = framework-arduinopico@https://github.com/earlephilhower/arduino-pico.git#6024e9a7e82a72e38dd90f42029ba3748835eb2e ; 4.3.0 with fix MDNS
+platform_packages = framework-arduinopico@https://github.com/earlephilhower/arduino-pico.git#b506c010f7766cfb181c8da6db93cb8d554460f6 ; 4.4.3
```

```suggestion
platform_packages = framework-arduinopico@https://github.com/earlephilhower/arduino-pico.git#4.4.3
```
You can also reference tags directly with PlatformIO. Much cleaner.